### ISSUE TITLE
Support visual mode when cmdremote enabled

### ISSUE THREAD

**Issue by radare** [radareorg/ideas#40]
We can probably copypasta the code from r2 -C as an initial version, but ideally, the whole visual.c should work with local and remote modes. !=! =!=

**Comment by trufae** [comment]
Yes it's relevant, because visual mode doesnt works with remote sessions, but there's an issue in ideas to improve the support for remoting in different situations.

This issue is related https://github.com/radareorg/ideas/issues/202
